pr_agent/algo/__init__.py

```diff
@@ -1,4 +1,6 @@
 MAX_TOKENS = {
+    'bito_basic': 12000,
+    'bito_advanced': 90000,
     'text-embedding-ada-002': 8000,
     'gpt-3.5-turbo': 4000,
     'gpt-3.5-turbo-0613': 4000,
```

Comment:
**Suggestion**: Consider adding comments explaining the rationale behind the new MAX_TOKENS values for 'bito_basic' and 'bito_advanced'. This will help maintainers understand the choice of values and their impact on the application's performance. <br> **Code Suggestion**: 
 ```
 +    'bito_basic': 12000, # Explanation for bito_basic value
 +    'bito_advanced': 90000, # Explanation for bito_advanced value
 ```